packages/appChrome/components/HeaderBar.tsx

```diff
@@ -10,20 +13,30 @@ export interface HeaderProps {
 class Header extends React.PureComponent<HeaderProps, {}> {
   public render() {
     const { children } = this.props;
+    /* tslint:disable:no-string-literal */
+    const HeaderBar = styled("div")`
+      background-color: ${props =>
+        props.theme.backgroundColor || purpleDarken4};
+      ${props =>
+        !props.theme.backgroundColor ||
+        (props.theme.backgroundColor && isHexDark(props.theme.backgroundColor))
+          ? darkMode
+          : null};
```

Comment:
A couple newbie emotion questions: Would `null;` be an issue here? Is there a benefit to having all these properties as functions instead of destructuring `theme` from props above with `children` and using them directly?

Reply:
These `props` aren't the same as the `this.props` above. Took me some time figuring out why `this.props.theme` didn't work.